tests/pos/inline-t9232b.scala

```diff
@@ -0,0 +1,23 @@
+final class Foo(val value: Int)
+
+object Foo {
+  inline def unapplySeq(foo: Foo): Some[Seq[Int]] = Some(List(foo.value))
```

Comment:
Should this be changed to `inline def unapply(foo: Foo)` after #3747?

Reply:
I would say not for now, if/when `unapplySeq` is effectively not supported in Dotty then we should turn this tests into a scala2 test.